src/devices/device_constructors/branch_constructor.jl

```diff
@@ -0,0 +1,28 @@
+function construct_device!(ps_m::CanonicalModel,
+                           device::Type{B},
+                           device_formulation::Type{Br},
+                           system_formulation::Type{StandardPTDFForm},
+                           sys::PSY.PowerSystem, 
+                           time_range::UnitRange{Int64};
+                           kwargs...) where {Br <: AbstractLineForm,
+                                             B <: PSY.Branch}
+
+    line_rate_constraints(ps_m, sys.branches, device_formulation, system_formulation, time_range)
+
+    return
+
+end
+
+function construct_device!(ps_m::CanonicalModel,
+                            device::Type{B},
+                            device_formulation::Type{Br},
+                            system_formulation::Type{S},
+                            sys::PSY.PowerSystem, 
+                            time_range::UnitRange{Int64};
+                            kwargs...) where {Br <: AbstractLineForm,
+                                              B <: PSY.Branch,
+                                              S <: PM.AbstractPowerFormulation}
+
+    return
```

Comment:
I'd add a comment or something saying that this is meant to do nothing.

Reply:
Fixed via a351de5